me.hyliu.fluentreader.appdata.xml

```diff
@@ -0,0 +1,49 @@
+
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Should be upstreamed.

Reply:
Unfortuantely they only use `electron-builder` and it haven't been able to handle appstream metadata https://github.com/electron-userland/electron-builder/issues/1993

Reply:
Technically true, but for the time being it could be maintained there and installed "manually" via build-commands or post-install.

Reply:
I'll have to ask the upstream developer to see where he would like to put it.

@yang991178